examdownloader-cli.py

```diff
@@ -32,5 +24,28 @@ def downloadCallback(status, lastfile='', numFiles=0):
 
     ed.getContents(module, username, password, destination, downloadCallback, updateStatus)
 
+
 if __name__ == '__main__':
-    startDownload(sys.argv[1:])
+    parser = argparse.ArgumentParser(description="Downloads exam papers from UCL automatically")
+    parser.add_argument('-u', '--user', help="NUSNET ID", type=str)
+    parser.add_argument('-d', '--dest', help="Path to store files", type=str)
+    parser.add_argument('-m', '--module', help="Module of which to download exam papers", type=str)
+    args = vars(parser.parse_args())
+
+    module = args.get("module")
+    if not username:
+        username = args.get("user")
+    if not username:
+        username = raw_input('Enter NUSNET ID: ')
+
+    password = getpass.getpass('Enter password for {}: '.format(username))
+
+    if not module:
+        module = raw_input("Enter module to download exams for: ")
+
+    if not destination:
+        destination = args.get("dest")
+    if not destination:
+        destination = raw_input('Enter location to store exams in: ')
```

Comment:
Let's use `.` as a default value for destination so that more novice users won't get confused

Reply:
Done